### ISSUE TITLE
nvda does not announce character selection after some time of use 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12119]
### Steps to reproduce:
use your nvda for 2 hours or so
now open a text field in chrome as an example the original text text field on translate.google.com
select some characters using the commands
ctrl + shift + end
shift arrows up and down 
### Actual behavior:
the nvda is silent and does not announce the selection, the selection is carried out despite the lack of speech by the nvda 
### Expected behavior:
nvda should announce text selection normally
Note that restarting nvdaa solves the problem 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-21810,db664bef

#### Windows version:
10.0.19043.844
#### Name and version of other software in use when reproducing the issue:
chrome.exe, Google Chrome 88.0.4324.190

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no
log:
[select.txt](https://github.com/nvaccess/nvda/files/6085605/select.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva please run the COM registration fixing tool in NVDA menu 7 tools. How are the results after that?

**Comment by ABuffEr** [comment]
Hi,
I had this issue too, solved running COM registration fixing tool.
But are there been changes that explain this behaviour? Because I'm noticing a lot of similar issues, even during input (delayed insert, duplicated characters...), and I have run COM fixing tool a lot of times in last 2-3 days.

**Comment by fernando-jose-silva** [comment]
thank you very much, I ran the registry correction tool with,
and unfortunately the problem continues to occur.

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still having this problem with NVDA 2024.1 Beta 9 or last alpha and Google Chrome 122?

**Comment by fernando-jose-silva** [comment]
In recent times I no longer see this problem, for me at this point we can close it.
